vignettes/symptom-surveys.Rmd

```diff
@@ -0,0 +1,672 @@
+    )
+  )$predictions %>%
+    mutate(lead = .x) %>%
+    left_join(z_te %>% filter(target_date == (date + .x)), by = c("geo_value", "target_date"))) %>%
+    list_rbind() %>%
+    mutate(err1 = abs(inv_trans(.pred) - inv_trans(target_case))) %>%
+    select(geo_value, forecast_date, err1, lead)
+
+  # Cases and Facebook model
+  out_df2 <- map(leads, ~ arx_forecaster(
+    z %>%
+      filter(between(time_value, date - .x - max(lags) - n, date)) %>%
+      select(time_value, geo_value, case, fb) %>%
+      filter(complete.cases(.)),
+    outcome = "case",
+    predictors = c("case", "fb"),
+    trainer = quantile_reg(),
+    args_list = arx_args_list(
+      lags = lags,
+      ahead = .x,
+      nonneg = FALSE
+    )
+  )$predictions %>%
+    mutate(lead = .x) %>%
+    left_join(z_te %>% filter(target_date == (date + .x)), by = c("geo_value", "target_date"))) %>%
+    list_rbind() %>%
+    mutate(err2 = abs(inv_trans(.pred) - inv_trans(target_case))) %>%
+    select(geo_value, forecast_date, err2, lead)
+
+
+  # Cases and Google model
+  out_df3 <- map(leads, ~ arx_forecaster(
+    z %>%
+      filter(between(time_value, date - .x - max(lags) - n, date)) %>%
+      select(time_value, geo_value, case, goog) %>%
+      filter(complete.cases(.)),
+    outcome = "case",
+    predictors = c("case", "goog"),
+    trainer = quantile_reg(),
+    args_list = arx_args_list(
+      lags = lags,
+      ahead = .x,
+      nonneg = FALSE
+    )
+  )$predictions %>%
+    mutate(lead = .x) %>%
+    left_join(z_te %>% filter(target_date == (date + .x)), by = c("geo_value", "target_date"))) %>%
+    list_rbind() %>%
+    mutate(err3 = abs(inv_trans(.pred) - inv_trans(target_case))) %>%
+    select(geo_value, forecast_date, err3, lead)
+
+  # Cases, Facebook and Google model
+  out_df4 <- map(leads, ~ arx_forecaster(
+    z %>%
+      filter(between(time_value, date - .x - max(lags) - n, date)) %>%
+      select(time_value, geo_value, case, fb, goog) %>%
+      filter(complete.cases(.)),
+    outcome = "case",
+    predictors = c("case", "goog"),
+    trainer = quantile_reg(),
+    args_list = arx_args_list(
+      lags = lags,
+      ahead = .x,
+      nonneg = FALSE
+    )
+  )$predictions %>%
+    mutate(lead = .x) %>%
+    left_join(z_te %>% filter(target_date == (date + .x)), by = c("geo_value", "target_date"))) %>%
+    list_rbind() %>%
+    mutate(err4 = abs(inv_trans(.pred) - inv_trans(target_case))) %>%
+    select(geo_value, forecast_date, err4, lead)
+
+  # Left join of the results for all models
+  out_list[[k]] <- left_join(left_join(left_join(left_join(out_df0, out_df1), out_df2), out_df3), out_df4)
+}
+# Outside of loop bind rows of list
+out_df <- do.call(rbind, out_list)
+```
+
+## Results: All Four Models
+
+Since there are only two common forecast dates available for the four models for the 14-day-ahead forecasts (May 13 and May 14, 2020), we skip studying the 14-day-ahead forecast results in this four-way model discussion. 
```

Comment:
Why is this?

Reply:
Paused running the Google survey on May 14 so its data ends there.